readyset-adapter/src/backend.rs

```diff
@@ -1715,11 +1715,18 @@ where
     async fn show_proxied_queries(
         &mut self,
         query_id: &Option<String>,
+        only_supported: &Option<bool>,
     ) -> ReadySetResult<noria_connector::QueryResult<'static>> {
         let mut queries = self.state.query_status_cache.deny_list();
         if let Some(q_id) = query_id {
             queries.retain(|q| &q.id.to_string() == q_id);
         }
+        if  let Some(supported) = only_supported {
```

Comment:
looks like you need to run `cargo fmt`

Reply:
Done. I am new to rust; I did not know about it.